test/get.coffee

```diff
@@ -255,7 +255,7 @@ describe 'quest', ->
       it 'supports changing the port', (done) ->
         @timeout 20000
         options =
-          uri: "#{protocol}://httpbin.org:81334/get"
+          uri: "#{protocol}://httpbin.org:65334/get"
```

Comment:
Without this, [drone fails](https://ci.ops.clever.com/github.com/Clever/quest/data-patch-1/8aab642b4e7b66fe9928d2d666889636c5969356) with the following error from the `net` native package: `RangeError: "port" option should be >= 0 and < 65536: 81334`